### PR TITLE
Update android bindings and add github action

### PR DESCRIPTION
**Pull request by lvaccaro**
- bindings: fix copy kotlin lib in just file
- bindings: disable gradle lint on android build
- ci: add kotlin github action

### CODE REVIEW

**Comment by lvaccaro** [comment]
rebased from master and update copy path in justfile

File: .github/workflows/kotlin.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+      - uses: dtolnay/rust-toolchain@1.75.0
+        with:
+          targets: x86_64-linux-android, aarch64-linux-android, armv7-linux-androideabi
+
+      - uses: extractions/setup-just@v1
+        with:
+          just-version: 1.5.0  # optional semver specification, otherwise latest
+      
+
```
Comment (RCasatta) [inline .github/workflows/kotlin.yml:34]: shouldn't `just kotlin-android` be called here?

**Comment by RCasatta** [comment]
utACK 9daa269f5e363d11e4a40aeecd75f3c27e5bb248 with passing ci

File: .github/workflows/kotlin.yml (hunk 1 above)
Comment (lvaccaro) [reply .github/workflows/kotlin.yml:41]: resolved thanks

**Comment by RCasatta** [comment]
utACK 9c933bf0ad46da8cee2a05d3b6a67d689c9a2135

**Review by RCasatta — APPROVED** [review]

**Pull request merged**